### PR TITLE
Configure script to generate the google play url

### LINKED ISSUE #129: Configure script to generate the google play url
**Is your feature request related to a problem? Please describe.**
Every radio has a link to the Google Play store. We're creating applications before they're even uploaded to the store. That said, the script should generate a proper url to use as link to the store.
The url is the store is generated based on the application id: (e.g.: com.myorg.myapp).

**Describe the solution you'd like**
The script could generate this url by reading the package name in the `google-services.json`. 
Google handles urls this way: `https://play.google.com/store/apps/details?id=<com.example.radio>`
The script should generate this url and expose it to the environment variable.

We'll need to add this attribute in the sample file created in #111. Then we must add it to the environment variables passed to webpack so that it can be used in `config.js` to fill the `shareApp` attribute.

### shareApp

+   Details: An object with information to share the app from Google Play
+   Type: `Object`
+   Structure:
     +   playStoreUrl: `String`
     +   message: `String`

### PR DESCRIPTION
**Pull request by estefi-prieto**
This PR provides:
 Google Play url to share app

Closes #129

### CODE REVIEW

File: app/langs/locales/es.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "writeUs": "Escríbenos",
+  "messageToShareApp": "Bajate la app",
```
Comment (sgobotta) [inline app/langs/locales/es.json]: `Descarga` would be a more generic word for spanish locale, what do you think @EstefiCamba  ?

File: tools/create-app.bash [hunk 2]
```diff
@@ -24,10 +24,10 @@ cp $GOOGLE_SERVICES_PATH $DIRECTORY_PATH/../app/App_Resources/Android/
 # Parse json configuration file to string
 STRINGIFIED_CONFIGURATION=$(jq '. | tostring' $CONFIGURATION_FILE)
 
-APP_NAME=$(jq '.name | tostring' $CONFIGURATION_FILE | tr -d "\"")
+APP_ID=$(jq '.client[0].client_info.android_client_info.package_name | tostring' $GOOGLE_SERVICES_PATH | tr -d "\"")
```
Comment (sgobotta) [inline tools/create-app.bash:27]: It seems `jq` is not a built in command in some linux operating systems. Did you happen to have done `sudo apt-get install jq`?

**Comment by sgobotta** [comment]
I'm having a hard time trying to customize the application.

This is what im doing:

+   I uninstall the app and all cached data from the app in my celphone

I used this config:

```json
{
  "name": "radio",
  "social": {
    "facebookUrl": "https://www.facebook.com/cambacoop/",
    "instagramUrl": "https://www.instagram.com/",
    "twitterUrl": "https://www.twitter.com/"
  },
  "colors": {
    "backgroundColor": "#AF002A",
    "primaryText": "#E500A2",
    "secondaryText": "#ffffff"
  },
  "bottomNavigationBar": {
    "activeColor": "#E500A2",
    "backgroundColor": "#1a121f",
    "inactiveColor": "#808080"
  },
  "stream": "https://",
  "panelBackgroundColor": {
    "backgroundColor": "#1a121f"
  },
  "phoneNumber": "",
  "sections": [
    "Live",
    "Schedule",
    "WriteUs",
    "Social"
  ],
  "writeEmailTo": [],
  "shareApp": {
    "playStoreUrl": "",
    "message": ""
  }
}
```
In the crate-app.sh i used:
+   a green background color (#00FF00) for the splash color parameter
+   change the splash screen png for a customised one (i override the ns-vue-radio logo)

Then I build an .aab and with that aab i install the app in my phone.

Result:

+ The splash background color is not applied. I don't know if it's related but the background looks like the one defined in config.js -> #AF002A
+ The splash image doesn't change at all: I see the ns-vue-radio logo, like nothing changed at all.

File: tools/create-app.bash (hunk 2 above)
Comment (sgobotta) [reply tools/create-app.bash:27]: I can add this dependency need to the script documentation

**Review by sgobotta — APPROVED** [review]

**Comment by sgobotta** [comment]
Ready to merge. A [ticket](https://github.com/Cambalab/ns-vue-radio/issues/148) has been opened regarding the splash screen bug

File: app/langs/locales/es.json (hunk 1 above)
Comment (estefi-prieto) [reply app/langs/locales/es.json]: I think that is the same, but if you prefer use 'descarga', we can use that

File: tools/create-app.bash (hunk 2 above)
Comment (estefi-prieto) [reply tools/create-app.bash:27]: > Did you happen to have done sudo apt-get install jq?

No, I didn't

> I can add this dependency need to the script documentation

great!

**Pull request merged**